admin_ui/src/components/CreateNewUser.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <div id="change_password">
+        <div class="inner">
+            <BackButton />
+            <h1>Change password</h1>
+            <form v-on:submit.prevent="changePassword">
+                <label>Old password</label>
+                <input
+                    name="old_username"
+                    type="password"
+                    v-model="oldPassword"
+                />
+
+                <label>New password</label>
+                <input
+                    name="new_password"
+                    type="password"
+                    v-model="newPassword"
+                />
+
+                <label>New password confirmation</label>
+                <input
+                    name="confirm_password"
+                    type="password"
+                    v-model="confirmPassword"
+                />
+
+                <button>Change password</button>
+            </form>
+        </div>
+    </div>
+</template>
+
+
+<script>
+import axios from "axios"
+import BackButton from "../components/BackButton.vue"
+
+export default {
+    data() {
+        return {
+            oldPassword: "",
+            newPassword: "",
+            confirmPassword: ""
+        }
+    },
+    components: {
+        BackButton
+    },
+    methods: {
+        async changePassword() {
+            console.log("Changing password")
+            const payload = {
+                old_password: this.oldPassword,
+                new_password: this.newPassword,
+                confirm_password: this.confirmPassword
+            }
+            try {
+                await axios.post(`./auth/change-password/`, payload)
+                this.$store.commit("updateApiResponseMessage", {
+                    contents: `Changed password successfully. You will be redirected 
+                        to the login page to log in with a new credentials.`,
+                    type: "success"
+                })
+                setTimeout(() => {
+                    this.$router.push({ name: "home" })
+                }, 3000)
+            } catch (error) {
+                console.log("Request failed")
+                console.log(error.response)
+                this.$store.commit("updateApiResponseMessage", {
+                    contents:
+                        "The form has errors. Changing password canceled.",
+                    type: "error"
+                })
+                setTimeout(() => {
+                    this.$router.push({ name: "home" })
+                }, 5)
+                return
+            }
+        }
+    }
+}
+</script>
+
```

Comment:
I guess this is a work in progress, as it uses the change password endpoint at the moment.

We need to be careful with letting the user register other users. We should only allow it if the user has `superuser=True`.

Reply:
Wrong copy/paste. I fixed that.

Reply:
Yeah. That makes sense. Thanks. I just have to change this.
```python
api_app.mount(
    path="/register/",
    app=AuthenticationMiddleware(
        register(redirect_to="./../../auth/login/"),
        SessionsAuthBackend(superuser_only=True),
    ),
)
```
I'll do it later today.

Reply:
Cool - thanks

Reply:
That's better. I think we should grey out and disable the 'create user' button in the UI if they're not super users. Otherwise they may try and create a user, and it will fail.

<img width="214" alt="Screenshot 2022-06-21 at 16 51 36" src="https://user-images.githubusercontent.com/350976/174843561-e90f983c-45b0-4381-8c45-e9ecb78d65db.png">

Reply:
I've added a commit for this which you can check out.